### ISSUE TITLE
A11y_WindowsUILibraryv2.4_HomeScreen_ListItemsPane_ScreenReader: Narrator announces incorrect numbering of list items.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/microsoft-ui-xaml#3143]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_WindowsUILibraryv2.4;#A11yTCS;#A11yMAS;#DesktopApp;#WCAG1.3.1;#Windows UI Library;#STP;#NVDA;#Win32;#Narrator;#FTP;#Benchmark;#A11ySev2

**Environment Details:**
Application Name: Windows UI Library v2.4
Version: Version 2.4
OS: Windows10

**Additional Details:**
MAS References:WCAG1.3.1

**Repro Steps:**

1.    Install and open the Windows UI Library v2.4 application
2.    Navigate to List Item pane on the left side of the screen.
3.    Navigate to "Radial Gradient brush" list item under "New Control" start narrator and check the announcement

**Actual:**
On navigating to "Radial Gradient brush" list item under "New Control" narrator announces it as 2 of 2 level 1 which is incorrect. It also announces same for "Progress Ring" list item. also same for "Tab view" and "Test box Family

**Issue also observed with NVDA**

**Expected:**
On navigating to "Radial Gradient brush" list item under "New Control" narrator should announce it as 1 of 2 level 1 similarly it should announce correct numbering for others.

**User Impact:**
Screen reader users will receive the wrong information.

**Attachment link for Reference:**

[A11y_WindowsUILibraryv2.4_HomeScreen_ListItemsPane_ScreenReader.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5149289/A11y_WindowsUILibraryv2.4_HomeScreen_ListItemsPane_ScreenReader.zip)

**Comment by StephenLPeters** [comment]
I think this is an issue with Hierarchical Nav View not reporting the correct size position and depth of set.  #2627 may have fixed this already? @YuliKl

**Comment by YuliKl** [comment]
I'm not able to repo the problem. The "radial gradient brush" item is being announced as "1 of 2, level 2" as expected.
